src/main/java/gameboy/gamepack/racinggame/data/vo/Position.java

```diff
@@ -1,37 +1,56 @@
 package gameboy.gamepack.racinggame.data.vo;
 
-public class Position {
+import java.util.Objects;
 
-    private int position;
+public class Position implements Comparable<Position> {
+
+    private static final int MIN_POSITION_VALUE = 0;
+
+    private int value;
 
     public Position() { }
 
-    public Position(int position) {
-        validationPosition(position);
-        this.position = position;
+    public Position(int value) {
+        validationPosition(value);
+        this.value = value;
     }
 
-    private void validationPosition(Integer position) {
-        if (position.intValue() <= 0) {
-            throw new IllegalArgumentException("0이하의 포지션 생성: " + position);
+    private void validationPosition(int position) {
+        if (position < MIN_POSITION_VALUE) {
+            throw new IllegalArgumentException(MIN_POSITION_VALUE + "미만의 포지션 생성: " + position);
         }
     }
 
-    public void add() {
-        position += 1;
+    public void addOne() {
+        value += 1;
     }
 
-    public int getPosition() {
-        return position;
+    public int value() {
+        return value;
     }
 
     @Override
-    public boolean equals(Object obj) {
-        return this.hashCode() == obj.hashCode();
+    public boolean equals(Object that) {
+        if (that == null || getClass() != that.getClass()) {
+            return false;
+        }
+        return Objects.equals(value, ((Position) that).value);
```

Comment:
자바16부터 나온 패턴매칭이라는 게 있습니다.
한번 검색해 보시고 학습해보시면 좋겠네요

```suggestion
    @Override
    public boolean equals(Object that) {
        if (this == that) {
            return true;
        }
        if (!(that instanceof Position position)) {
            return false;
        }
        return this.value == position.value;
    }
```